yb-voyager/cmd/import.go

```diff
@@ -75,16 +72,15 @@ func registerCommonImportFlags(cmd *cobra.Command) {
 
 	cmd.Flags().StringVar(&target.User, "target-db-user", "",
 		"Username with which to connect to the target YugabyteDB server")
+	cmd.MarkFlagRequired("target-db-user")
```

Comment:
> Have you tested whether keeping the flags' checks here will change the functioning of the code? It may be better yet to keep these checks within the `validateImportFlags()` function for uniformity

the motive here is to keep all the statements like `cmd.MarkFlagRequired()` alongside with the flag side wherever possible, as a best practice.